guardian/support-frontend
# Guardian Ad-Lite: returnAddress searchParam pass through

## What are you doing in this PR?

From the Guardian Ad-Lite Landing Page Onwards...

With returnAddress supplied->
1. LP: Accept a paramUrl  `returnAddress`
2. LP: Link Accept All button through to the Url in` returnAddress`
3. LP: Route paramUrl  `returnAddress` onto Checkout
4. Checkout: Link javascript.back() button so ` returnAddress` appended
5. Checkout: Route paramUrl  `returnAddress` into Thankyou page

Without returnAddress supplied (defaultUrl= `https://www.theguardian.com/${geoId}`->
1. LP: missing paramUrl  `returnAddress`, link Accept All button through to the default Url
2. Checkout: Link javascript.back() button so ` returnAddress` appended
3. ThankYou: missing paramUrl  `returnAddress`,  link to the default Url

NOTE: Thankyou page waiting for construction.


[**Trello Card**](https://trello.com/c/RlDRXr5W/1308-cop-add-returnaddress-paramurl-throughout-journey)

## How to test

- append url to returnAddress ParamURL, check Url routes through teh checkout and onto thankyou pages
https://support.thegulocal.com/uk/guardian-light?returnAddress=https%3A%2F%2Fwww.theguardian.com%2Fuk
https://support.thegulocal.com/uk/guardian-light
